CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@
 * [Capturing ESX Code Coverage](https://github.com/vmware/docker-volume-vsphere/blob/master/CONTRIBUTING.md#capturing-esx-code-coverage)
```

Comment:
`[git, branch management and pull requests](` <--- this is not needed it should be part of "Typical dev workflow"